lib/msal-common/src/telemetry/performance/IPerformanceClient.ts

```diff
@@ -21,6 +21,7 @@ export type InProgressPerformanceEvent = {
 export interface IPerformanceClient {
     startMeasurement(measureName: PerformanceEvents, correlationId?: string): InProgressPerformanceEvent;
     endMeasurement(event: PerformanceEvent): PerformanceEvent | null;
+    // @deprecated use "endMeasurement" instead
```

Comment:
This is OK for a patch version?

Reply:
Looks like it isn't going away so this is likely OK

Reply:
I will mark a consequent PR that removes `flushMeasurement` method as minor.